SU-SWS/stanford_media
# D8CORE-2753: convert to webpack and caption edits.

# READY FOR REVIEW

# Summary
- Fix the over run padding on media added to the wysiwyg.
- convert from grunt to webpack

# Needed By (Date)
- 11/20 - so I can make edits before the break.

# Urgency
- Medium

# Steps to Test

1. Pull in this change.
2. Check the webpack changes. 
3. Check the admin changes.
4. Add media to a text paragraph. Try different sizes and alignments. Try a video. (I've tried all of these and tried it on SOE)
5. Verify that the caption has the same left and right as the image. The caption is centered.  Note: the Media Paragraph has a right aligned caption.

# Affected Projects or Products
- All products using the Text paragraph with media/captions

# Associated Issues and/or People
- D8CORE-2753

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
